src/utils/verify.test.ts

```diff
@@ -75,6 +75,6 @@ describe('#verify.testProjects()', () => {
 
 		const config = await getConfig();
 
-		expect(verify.testProjects(config)).toMatch(/\/rugged\/test\-file\-trees\/primary\/test\-projects$/);
+		expect(verify.testProjects(config)).toMatch(/(\/|\\)rugged\1test\-file\-trees\1primary\1test\-projects$/);
```

Comment:
@corymharper Can you explain this change? And is there a way to make this more readable, possibly by using `path.join()` instead of writing the string directly?

Reply:
Sure, Windows paths just use a backward slash instead of a forward slash. The rest of the regex is just referencing the first capture group to make it more succinct, instead of repeating `(\/|\\)` several times.